tests/queries/0_stateless/03312_system_errors_last_error_message.reference

```diff
@@ -0,0 +1 @@
+Value passed to \'throwIf\' function is non-zero: while executing \'FUNCTION throwIf(1_UInt8 :: 1) -> throwIf(1_UInt8) UInt8 : 2\'
```

Comment:
Before this PR's changes the `last_error_message` here was just `Value passed to \'throwIf\' function is non-zero`